file_organizer_v2/src/file_organizer/cli/profile.py

```
@@ -0,0 +1,509 @@
+        if profile is None:
+            click.echo(f"Failed to create profile '{name}'", err=True)
+            raise click.Abort()
+
+        click.echo(f"✓ Created profile: {name}")
+
+        # Activate if requested
+        if activate:
+            if manager.activate_profile(name):
+                click.echo(f"✓ Activated profile: {name}")
+            else:
+                click.echo(f"✗ Failed to activate profile", err=True)
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)